core/controllers/topics_and_skills_dashboard.py

```diff
@@ -110,6 +113,7 @@ def get(self):
             'topic_summary_dicts': topic_summary_dicts,
             'can_delete_topic': can_delete_topic,
             'can_create_topic': can_create_topic,
+            'can_delete_skill': can_delete_skill,
```

Comment:
Maybe modify some existing tests to make sure this is returned correctly?

Reply:
Done